.github/workflows/test-phpstan.yml

```diff
@@ -34,7 +34,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php-versions: ['8.0', '8.1']
+        php-versions: ['8.1']
```

Comment:
Remove `matrix` then put php-version in L45.

Reply:
Done.